check/tests/pass.rs

```diff
@@ -436,9 +432,9 @@ let function_test: Test ((->) a) = {
 function_test.test
 ";
     let result = support::typecheck(text);
-    let expected = Ok(Type::function(vec![typ("a")], typ("Int")));
+    let expected = Ok("forall a . a -> Int".to_string());
 
-    assert_req!(result, expected);
+    assert_req!(result.map(|typ| typ.to_string()), expected);
```

Comment:
These stringified tests are much easier to read! 👏

Reply:
They are however not as accurate :/ Not a problem in the tests in question though.

Probably better to do it the other way around and parse a string into the type representation. Where that is possible at least.